### PR TITLE
Add Azure Quantum team to code owners

### PR DESCRIPTION
**Pull request by guenp**

### CODE REVIEW

**Comment by guenp** [comment]
ping @joshfree , would you please have a look? thanks!

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -122,6 +122,9 @@
 /sdk/**/azure-resourcemanager-*                     @ChenTanyi @weidongxu-microsoft @yungezz
 /sdk/**/mgmt*/                                      @ChenTanyi @weidongxu-microsoft @yungezz
 
+# Quantum
```
Comment (srnagar) [inline .github/CODEOWNERS:125]: Include a PR label too. If this label does not exist, please create one.
```
# PRLabel: %Azure Quantum
```

**Comment by guenp** [comment]
Hi @jsquire  and @srnagar! I am having some problems triggering, the check-enforcer, it seems to be stuck. Would you have a look and see why it is not running? Thanks!

**Review by JonathanGiles — APPROVED** [review]

**Pull request merged**